.circleci/config.yml

```diff
@@ -70,7 +70,7 @@ commands:
           command: |
             source $BASH_ENV
             source $HOME/venv/bin/activate
-            git checkout ${<< parameters.tridenttag >>}
+            git checkout ${<< parameters.tridenttag >>} || git checkout $TRIDENT_HEAD
```

Comment:
With this change, we won't have to first set `TRIDENT_GOLD` to be `HEAD` and then change it with a direct commit after the pull request is merged. We'll still have to remember to add the tag and push it.